app/furniture/marketplace/tags/index.html.erb

```diff
@@ -1,16 +1,47 @@
 <%- breadcrumb :marketplace_tags, marketplace %>
-
 <%= render CardComponent.new do |card| %>
-
-  <%- marketplace.tags.each do |tag| %>
-    <%- tag.marketplace = marketplace %>
-    <div id="<%= dom_id(tag)%>" class="flex flex-row gap-3">
-      <span class="flex-grow">
-        <%= tag.label %>
-      </span>
-    </div>
+  <h1>Product Groups</h1>
+  <p>Drag and drop to change the order in which groups will display on your Marketplace</p>
+  <ul data-controller="sortable" data-sortable-animation-value="150" data-sortable-resource-name-value="tag" class="flex flex-col gap-2 p-0">
+    <%- bazaar.tags.ordered_tag_groups.each do |tag| %>
+      <%-
+      # We must assign the marketplace object explicitly in the current scope
+      # because of the funky way we relate Tags to the Marketplace through a Bazaar.
+      # Yet our routing and navigation depends on a `marketplace` object being present
+      # on a Tag.
+      tag.marketplace = marketplace
```

Comment:
I don't understand this. I see that a Tag has an accessor for a Marketplace, but doesn't seem to have a direct association to one, and yet has a `location(parent: :marketplace)`, which is probably causing the need for this weird assignment here.

Seems like we are missing a `/bazaar` route namespace, and it is leading to this funkiness. If that's what we need, can't we just... add it?

Reply:
Yea, I think the path @rosschapman suggested to ditch the `bazaar` relationship on tags is the right one; which I think was your suggestion earlier that I stubbornly refused.